pages/index.tsx

```diff
@@ -107,6 +110,18 @@ export default function Home(props: {
 
 export const getServerSideProps: GetServerSideProps = async (context) => {
   const protocol = context.req.headers.referer?.split('://')[0] || 'http';
+  // const { data: keynoteData } = await RequestHelper.get<KeynoteSpeaker[]>(
+  //   `${protocol}://${context.req.headers.host}/api/keynotespeakers`,
+  //   {},
+  // );
+  const { data: challengeData } = await RequestHelper.get<Challenge[]>(
+    `${protocol}://${context.req.headers.host}/api/challenges/`,
+    {},
+  );
+  const { data: prizeData } = await RequestHelper.get<Array<{ rank: number; prizeName: string }>>(
+    `${protocol}://${context.req.headers.host}/api/prizes`,
+    {},
+  );
```

Comment:
I don't think we actually need this?

Reply:
where are we pulling the prize data from?